mcodingbot/plugins/python.py

```diff
@@ -7,7 +7,7 @@
 
 plugin = Plugin()
 
-PEP_REGEX = re.compile(r"pep *(?P<pep>\d{1,4})", re.IGNORECASE)
+PEP_REGEX = re.compile(r"pep[\s-]*(?P<pep>\d+)", re.IGNORECASE)
```

Comment:
why `+` instead of {1-4}?

Reply:
<img width="662" alt="Screen Shot 2022-09-13 at 00 16 41" src="https://user-images.githubusercontent.com/77130613/189768164-e530446e-e7da-466e-b4bc-0572f76678c1.png">
(pasting here for context)